scripts/apps/authoring/metadata/metadata.ts

```diff
@@ -752,7 +752,13 @@ function MetaTermsDirective(metadata, $filter, $timeout, preferencesService, des
                 scope.activeList = false;
             };
 
-            scope.isSelected = (term) => !!_.find(scope.item[scope.field], term);
+            scope.isSelected = (term) => {
+                if (scope.cv) {
+                    term.scheme = scope.cv._id;
```

Comment:
is `term.scheme` a name you came up now or is it used elsewhere? If not, I'd say `term.cvId` would be easier to understand

Reply:
it's used in `scope.item[scope.field]`

Reply:
only [here](https://user-images.githubusercontent.com/2076953/78659200-80a1c100-78cb-11ea-8cf6-46265f1a52be.png) or anywhere else too?

Reply:
There are more cases of it.
<img width="902" alt="Screenshot 2020-04-07 at 12 46 00" src="https://user-images.githubusercontent.com/4199922/78660672-ce1f2d80-78cd-11ea-9221-f94b489fcc28.png">

Reply:
got it, let's keep the name then. What is a bit strange is why are you writing data in an `isSelected` method which looks like it should only be reading data to determine if an item is selected.

How about writing this scheme field to all terms when initializing all the terms?

Reply:
That makes sense, I check if I can achieve it.